src/hotspot/cpu/aarch64/stubGenerator_aarch64.cpp

```diff
@@ -4690,25 +4689,25 @@ class StubGenerator: public StubCodeGenerator {
     const Register x     = r0;
     const Register xlen  = r1;
     const Register z     = r2;
-    const Register zlen  = r3;
```

Comment:
LibraryCallKit::inline_squareToLen() is still computing zlen and passing it as the 4th arg, even though the value is unused.

Reply:
OK.